libs/blocks/text/text.css

```diff
@@ -108,7 +108,79 @@
   padding-left: var(--spacing-xs);;
 }
 
+/* Legal */
+.legal.text-block .foreground {
+  padding: var(--spacing-xl) 0;
+}
+
+.legal.text-block p {
+  font-style: italic;
+  font-size: var(--type-body-xxs-size);
+  line-height: var(--type-body-xxs-lh);
+}
+
+/* link-farm */
+.link-farm.text-block {
+  padding: var(--spacing-xl) 0;
+}
+
+.link-farm.text-block .foreground {
+  display: grid;
+  grid-template-columns: auto;
+  gap: var(--spacing-xl) var(--spacing-m);
+}
+
+.link-farm.text-block h3 {
+  margin: 0 0 var(--spacing-l) 0;
```

Comment:
nit: the fourth value is not really needed, it will still be 0 if you have `margin: 0 0 var(--spacing-l);`

Reply:
Made changes accordingly.

Reply:
The margin is still showing as `0 0 var(--spacing-l) 0`

Reply:
Made changes.